File: @LV1/1.3/everywhere/everywhere.go
package main

import "fmt"

func main() {
	var n, m int
	var s string
	fmt.Scanf("%d", &n)

	for i := 0; i < n; i++ {
		var cnt int
		set := make(map[string]bool)

		fmt.Scanf("%d", &m)
		for j := 0; j < m; j++ {
			fmt.Scanln(&s)
			if !set[s] {
				set[s] = true
				cnt++
			}
		}
		fmt.Println(cnt)
	}
}

File: LV1/1.3/everywhere/everywhere.cpp
#include <cstdio>
#include <map>
#include <string>
using namespace std;

int main() {
    int n, m;
    char s[1000];
    map<string, bool> st;

    scanf("%d", &n);

    for (int i = 0; i < n; i++) {
        int cnt = 0;
        st.clear();
        scanf("%d", &m);

        for (int j = 0; j < m; j++) {
            scanf("%s", s);
            if (!st[s]) {
                st[s] = true;
                cnt++;
            }
        }
        printf("%d\n", cnt);
    }
}